conda/core/package_cache.py

```diff
@@ -96,6 +97,18 @@ def is_extracted(self):
         return isdir(epd) and isfile(join(epd, 'info', 'index.json'))
 
     @property
+    @memoizemethod
+    def repodata_record(self):
```

Comment:
We should fix that then.  Either in this PR or a future PR.  Actually, I'd say in a future PR, to make this one safer to cherry-pick back to 4.3.x if we decide to go that route.